string net_param # Serialized net parameter with weights (equivalent of prototxt file)
float32[] bias
float32[] scale
float32[] noise
int32 dim_bias
uint32 dU
